src/Runtime/OMUnique.inc

```diff
@@ -94,7 +94,8 @@ int isLessNum(void *arg1, void *arg2, OM_DATA_TYPE dataType) {
     return *((int32_t *)arg1) < *((int32_t *)arg2);
   case ONNX_TYPE_INT64:
     return *((int64_t *)arg1) < *((int64_t *)arg2);
-  // case ONNX_TYPE_STRING:
+  case ONNX_TYPE_STRING:
```

Comment:
I updated the missing `ONNX_TYPE_STRING` case for `[OMUnique.inc](https://github.com/onnx/onnx-mlir/pull/2478/files#diff-6406fc477efda850ee50a9eb2ba96b0a6edb3ce3e5de8b03072688c8ee29f146)`